HerPortal.BusinessLogic/Models/ConsortiumData.cs

```diff
@@ -1,7 +1,13 @@
 ﻿namespace HerPortal.BusinessLogic.Models;
 
+/// <summary>
+///     This class does not contain full Consortium Data.
+///     The full data (including an the Consortium's relationship to LAs) can be found in the HUG2 Public Website codebase
+/// </summary>
```

Comment:
I think it does contain all the data we have on consortia (just the names)? I suppose the more interesting points are that:
1. It's expected to match what's in the other repo
2. The mapping is in `LocalAuthorityData`